Advance_Python/Property.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
+
+#* ------------------------- Using @property decorator ------------------------ #
+
+class Celsius:
+    def __init__(self, temperature=0):
+        self.temperature = temperature
+
+    def to_fahrenheit(self):
+        return (self.temperature * 1.8) + 32
+
+    @property
+    def temperature(self):
+        print("Getting value...")
+        return self._temperature
+
+    @temperature.setter
```

Comment:
what exactly this method is called for???

Reply:
This method is decorated with Property thus whenever a value is setted it goes to fset()